### ISSUE TITLE
Clean up test folder

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#78]
It would be nice to standardize the file names and clean up the test folder. I like the format date-location-office. Maybe we could consistently prepend cvr_* and config_* to those.

There also might be some test files in there that aren't needed anymore or, if they are, could also have more consistent names (e.g. "minimum-threshold-test-cvr.xlsx", "test_continue_tabulation_config.json", etc.).

**Comment by HEdingfield** [comment]
Re-opening because it seems like tests added since this was completed don't follow these standards. In particular, should address:

- Everything that's run in `TabulatorTests.java` should follow the consistent `test_*` format (I see lots of folders with `*_test` and some that don't even have the word `test`.
- It seems like some CVR files, like `2013_minneapolis_park_cvr.xlsx` are used multiple times (e.g. `2013_minneapolis_park_hare_cvr.xlsx`, `2013_minneapolis_park_bottoms_up_cvr.xlsx`, `2013_minneapolis_park_sequential_cvr.xlsx` all appear to be the exact same file; @tarheel please confirm); maybe we should have a single copy and each of those test configs reference the same location?
- This is a minor / partial blocker to #281, because I think we need to distinguish between files that are included as regression tests, and those that are there for manual testing / playing with by users. Perhaps the latter type should be stored in a separate directory.
- Which leads to a final thing we should add: a setup that lets users easily trigger / play with the interactive tiebreaker.

**Comment by tarheel** [comment]
Yes, we do have a number of duplicate CVR files in the test data. If we wanted to avoid that, we would need to create an exception to the naming scheme, I guess. Also, 2013_minneapolis_mayor_scale has 13 copies of the same file because we don't allow a config to list the same source file multiple times. I agree it's dumb to have duplicate files in the repo, though.  Feel free to address these issues in a way that you think is reasonable.

**Comment by moldover** [comment]
It's not worth it to come up with a scheme to exclude a mere 25MB of test files.  If we make a distro which includes test files, we should include them all.

**Comment by HEdingfield** [comment]
Began the process of distinguishing between "test" and "sample" input files in PR #316. Verified that, in that PR, only the `sample_interactive_tiebreak` folder isn't used in a unit test, so all other folders should start with the prefix "test".
